src/init.cpp

```diff
@@ -1717,9 +1717,8 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
         bool minerAddressInLocalWallet = false;
         if (pwalletMain) {
             // Address has alreday been validated
-            CBitcoinAddress addr(mapArgs["-mineraddress"]);
-            CKeyID keyID;
-            addr.GetKeyID(keyID);
+            CTxDestination addr = DecodeDestination(mapArgs["-mineraddress"]);
```

Comment:
If `-mineraddress` has not been set, will the parameter be an empty string (the type expected) or null ptr?

Reply:
This is scoped inside `if (mapArgs.count("-mineraddress")) {`, so it will always be set here.